sklearn/linear_model/tests/test_base.py

```diff
@@ -195,9 +198,9 @@ def test_deprecate_normalize(normalize, default):
     assert _normalize == output
 
 
-def test_linear_regression_sparse(random_state=0):
+def test_linear_regression_sparse(global_random_seed):
     # Test that linear regression also works with sparse data
-    random_state = check_random_state(random_state)
+    random_state = check_random_state(global_random_seed)
```

Comment:
Test passes with all seeds. The `check_random_state` caught my eye. Should we replace all `np.random.RandomState(seed)` calls in `test_base.py` with `check_random_state(seed)`?